app/src/main/java/com/example/next_show/data/RecommendationClient.java

```diff
@@ -42,6 +67,11 @@ public void onResponse(Call<List<com.uwetrottmann.trakt5.entities.Show>> call, R
                         callback.onSuccess(Show.formatShows(response.body()));
                     } else {
                         callback.onFailure(response.code());
+
+                        // only if this first call fails...
+                        // get recommended shows based on User preferences -> updates adapter within callback
+                        fetchGenreMatchedShows(showsObj, genreMatchedCallback);
```

Comment:
Why would you want to end up doing this?  What errors are expected where this would fail and you'd want to make this fetch?  I would recommend pulling this out and handling it in the feed fragment, if the feed fragment wants to now try to fetch genre matched shows if related shows fail